examples/Flowchart.elm

```diff
@@ -14,129 +15,151 @@ import List exposing (head)
 
 
 type Flow
-  = Finish
-  | Task String
-  | Sequence Flow Flow
-  | Choice String Flow Flow
-  | Parallel (List Flow)
+    = Finish
+    | Task String
+    | Sequence Flow Flow
+    | Choice String Flow Flow
+    | Parallel (List Flow)
 
 
 example : Flow
 example =
-  Sequence
-    (Sequence (Task "check diff")
-      (Choice "diff is as whished"
-        (Sequence
-          (Parallel
-            [ Task "prepare changelog"
-            , Task "bump version"
-            ]
-          )
-          (Task "publish")
+    Sequence
+        (Sequence (Task "check diff")
+            (Choice "diff is as wished"
+                (Sequence
+                    (Parallel
+                        [ Task "prepare changelog"
+                        , Task "bump version"
+                        ]
+                    )
+                    (Task "publish")
+                )
+                (Task "work harder")
+            )
         )
-        (Task "work harder")
-      )
-    )
-    Finish
+        Finish
 
 
 
 -- Elements --------------------------------------------------------------------
 
 
+fontFamily : String
+fontFamily =
+    "Caveat"
```

Comment:
Better:
```suggestion
font : Font
font =
    Font "Caveat"
```

Reply:
Problem is I'm using this string to load the Google font below. Pretty hacky, let me know if I should take that out.

Reply:
Yeah, there is not a real nice way to solve this, isn't it? Maybe use both?

```elm
fontFamily : String
fontFamily =
    "Caveat"

font : Font
font =
    Font fontFamily
```